src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SyntaxFacts/CSharpSyntaxFacts.cs

```diff
@@ -813,72 +811,72 @@ public string GetDisplayName(SyntaxNode? node, DisplayNameOptions options, strin
                 return GetName(((BaseNamespaceDeclarationSyntax)node).Name, options);
             case SyntaxKind.QualifiedName:
                 var qualified = (QualifiedNameSyntax)node;
-                return GetName(qualified.Left, options) + dotToken + GetName(qualified.Right, options);
+                return $"{GetName(qualified.Left, options)}.{GetName(qualified.Right, options)}";
         }
 
-        string? name = null;
         if (node is MemberDeclarationSyntax memberDeclaration)
         {
-            if (memberDeclaration.Kind() == SyntaxKind.ConversionOperatorDeclaration)
+            if (memberDeclaration is ConversionOperatorDeclarationSyntax conversionOperator)
+                return Instance.ConvertToSingleLine(conversionOperator.Type).ToString();
```

Comment:
>Instance [](http://example.com/codeflow?start=23&length=8)

nit: I found it odd to go through Instance for this. just calling ConvertToSingleLine